ggml/src/ggml-webgpu/ggml-webgpu.cpp

```diff
@@ -858,6 +860,48 @@ static webgpu_command ggml_webgpu_mul_mat(webgpu_context & ctx,
     return ggml_backend_webgpu_build(ctx, ctx->mul_mat_pipeline[src0->type][src1->type], params, entries, wg_x);
 }
 
+static webgpu_command ggml_webgpu_unary_op(webgpu_context &              ctx,
+                                           ggml_tensor *                 src,
+                                           ggml_tensor *                 dst,
+                                           webgpu_pipeline &             pipeline,
+                                           bool                          in_place,
+                                           const std::vector<uint32_t> & xielu_params = {}) {
```

Comment:
instead of calling this `xielu_params`, call it `extra_params`, so it's more general for other future unary operators